README.md

```diff
@@ -803,6 +813,65 @@ $ ulimit -u # processes / threads
 
 Just pass a new number as the argument to change it.
 
+## Prometheus Support
+
+Vegeta has a built-in Prometheus Exporter that may be enabled during "attacks" so that you can point any Prometheus instance to Vegeta instances and get some metrics about http requests performance and about Vegeta process itself.
+
+To enable Prometheus Exporter on command line, see flags with "prom-" prefix.
+To enable Prometheus Exporter on lib usage, add Option "PrometheusEnable" and/or "PrometheusSettings".
+
+Prometheus HTTP endpoint will be available only during the lifespan of an "attack" and will be closed right after the attack is finished.
+
+The following metrics are exposed:
+
+* `request_bytes_in` - bytes count received from targeted servers by "url", "method" and "status"
+* `request_bytes_out` - bytes count sent to targeted server by "url", "method" and "status"
+* `request_seconds` - histogram with request latency and counters by "url", "method" and "status"
```

Comment:
Can we derive actual request rate from these metrics so that we could plot it or do we need to capture it separately?

Reply:
Thoughts?

Reply:
@flaviostutz: Pending input here.

Reply:
With prometheus you typically keep metrics very raw as counters and derive rates from them using PromQL.

This way if you had multiple Vegeta's running at once you could calculate the rates in aggregate or individually across multiple pods.

Reply:
Yes you are right. Rate is done by the Prometheus server while it queries the timeseries database after data ingestion.

(Nice that this project is being reactivated!)